runtime/tchannel_logger.go

```diff
@@ -119,3 +121,20 @@ func (l TChannelLogger) WithFields(fields ...tchannel.LogField) tchannel.Logger
 	}
 	return TChannelLogger{logger: l.logger.With(zfields...)}
 }
+
+// LogError logs warnings for timeout errors
+func LogError(logger *zap.Logger, err error, msg string) {
+	OnError(logger, err, msg, msg)
+}
+
+// OnError logs warnings for timeout errors
+func OnError(logger *zap.Logger, err error, warnMsg, errMsg string) {
```

Comment:
Add a `TODO` here. We want to improve the classification of errors to be something like : 

https://github.com/uber/tchannel-node/blob/master/errors.js#L907-L930

Reply:
Done.